core/drivers/imx_csu.c

```diff
@@ -72,11 +72,12 @@ const struct csu_setting csu_setting_imx7ds[] = {
 	{(-1), 0},
 };
 
-/* Set all masters to non-secure except the Cortex-A7 */
+/* Set all masters to non-secure except the Cortex-A cores */
+const struct csu_sa_setting csu_sa_imx6 = { 0x15554554, 0x2aaa8aa8 };
```

Comment:
Here don't we need to lock the settings also ?
Locking in case of imx7ds and imx6ul

Reply:
I didn't understand the question, but while writing this answer realized what was meant.
Keeping the explanation here. The first block:
```
0x15554554                                                                                                                                                                                    
 28  24  20  16  12   8   4   0
0b10101010101010100010101010100
```
Sets all non-TrustZone aware masters to non-secure except the Cortex-A cores, which is similar to i.MX6ull & imx7ds.
The second block:
```
0x2aaa8aa8
  28  24  20  16  12   8   4   0
0b101010101010101000101010101000
```
than sets the lock bits for every master.

After writing this I guess you meant to ask whether we want to lock the settings for Bit 0, the cortex-A cores?
I'll change that.

Reply:
Yes, you got it right.